.travis.yml

```diff
@@ -1,6 +1,11 @@
+language: python
+
 sudo: false
 
-language: python
-install: pip install doc8
+install: 
+ - pip install doc8
+ - npm install -g write-good
+
 script:
-    - doc8 README.rst
+ - doc8 README.rst
+ - write-good README.rst --weasel --so --thereIs --adverb --cliches
```

Comment:
Can you keep sub sections indented 4 spaces, it makes it easier to read in my experience.